persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -240,7 +240,7 @@ WHERE
     OR (type IS 'ONE_ON_ONE' AND userDeleted = 1) -- show deleted 1:1 convos, to maintain prev, logic
     )
     AND (protocol IS 'PROTEUS' OR (protocol IS 'MLS' AND mls_group_state IS 'ESTABLISHED'))
-    AND archived = ?
+    AND (archived = 0 OR archived = :isArchived)
```

Comment:
Good catch ! Don't we need a migration file as well to recreate the view ?

Reply:
No need imo, as the previous query didn't make it to any release yet in any case :)